gslib/tests/test_parallel_cp.py

```diff
@@ -124,6 +125,8 @@ def testCopyingDirContainingOneFileToBucket(self):
     lines = self.AssertNObjectsInBucket(dst_bucket_uri, 1)
     self.assertEqual(suri(dst_bucket_uri, 'dir1', 'foo'), lines[0])
 
+  @SkipForS3('Boto lib required for S3 does not handle paths '
+             'starting with slash.')
```

Comment:
For posterity you should note the same thing you do in test_rm.py, that V4 slashes preclude paths that start with slash.

Reply:
Fixed